auxin_cli/Cargo.toml

```diff
@@ -47,7 +47,7 @@ tracing-subscriber = "0.2.25"
 tracing-futures = { version = "0.2.5", features = ["futures-03", "tokio"] }
 
 tokio-native-tls = "0.3.0"
-tokio = { version = "1.6.1", features = ["full"] }
+tokio = { version = "1.16.1", features = ["full"] }
```

Comment:
needed for oneshot blocking_recv

Reply:
I approve of this version bump, sounds good.